crypto/Pos/Crypto/Signing/Types/Safe.hs

```diff
@@ -39,6 +39,13 @@ instance Show EncryptedSecretKey where
 instance B.Buildable EncryptedSecretKey where
     build _ = "<encrypted key>"
 
+instance Eq CC.XPrv where
```

Comment:
Why did you decide to make an orphan instance instead of defining it in `Cardano.Crypto.Wallet`?

Reply:
AFAIK we are not supposed to touch code that comes from external git dependency

Reply:
I don't see why.

Reply:
At least change it to `deriving instance Eq CC.XPrv` and write a TODO about adding this instance to `Cardano.Crypto.Wallet` (I don't like unnecessary orphan instances)

Reply:
I did that with todo about what needs to be added to `Cardano.Crypto.Wallet`